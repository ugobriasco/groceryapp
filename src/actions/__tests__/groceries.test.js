import {
	getInitialGroceries, 
	GET_INITIAL_GROCERIES,
	updateGroceriesView,
	UPDATE_GROCERIESVIEW,
	updateGroceries,
	UPDATE_GROCERIES,

} from '../groceries';

it('create a GET_INITIAL_GROCERIES action', () => {
	expect(getInitialGroceries()).toMatchSnapshot();
});

it('create a UPDATE_GROCERIESVIEW action', () => {
	expect(updateGroceriesView([{id:'1'}])).toMatchSnapshot();
});

it('create a UPDATE_GROCERIES action', () => {
	expect(updateGroceries([{id:'1'}])).toMatchSnapshot();
});



