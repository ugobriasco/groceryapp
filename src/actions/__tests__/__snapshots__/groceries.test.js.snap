// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a GET_INITIAL_GROCERIES action 1`] = `
Object {
  "type": "GET_INITIAL_GROCERIES",
}
`;

exports[`create a UPDATE_GROCERIES action 1`] = `
Object {
  "type": "UPDATE_GROCERIES",
  "updatedGroceries": Array [
    Object {
      "id": "1",
    },
  ],
}
`;

exports[`create a UPDATE_GROCERIESVIEW action 1`] = `
Object {
  "type": "UPDATE_GROCERIESVIEW",
  "updatedGroceriesView": Array [
    Object {
      "id": "1",
    },
  ],
}
`;
